yarn.lock

```diff
@@ -0,0 +1,12273 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I think this also should be purged from the history along with package-lock.json. And we should probably remove the yarn note since there is the `npm run dev-server` option already? Maybe that needs to be documented?